openfisca_us/variables/demographic/geographic/county_str.py

```diff
@@ -0,0 +1,12 @@
+from openfisca_us.model_api import *
+
+
+class county_str(Variable):
+    value_type = str
+    entity = Household
+    label = "County (string)"
+    documentation = "County variable, stored as a string"
+    definition_period = YEAR
```

Comment:
I think this will fix the test issue. Otherwise try changing `ETERNITY` to `2022` in the test.
```suggestion
    definition_period = ETERNITY
```